cgrok/pcre_internal.h

```diff
@@ -310,7 +310,7 @@ defined here.
 The macros are controlled by the value of LINK_SIZE. This defaults to 2 in
 the config.h file, but can be overridden by using -D on the command line. This
 is automated on Unix systems via the "configure" command. */
-
+#define LINK_SIZE 2
```

Comment:
It was necessary to set this here, it didn't seem to work when it was exported in `config.h`. Not 100% sure why.